### PR TITLE
Reduce size of reoccurring query report

### LINKED ISSUE #310: Reduce size of query hash report table
Streamlit on snowflake is limted to 2MB per message and the query has report table was much larger (some very long queries) this produced an ugly error messag.

### PR DESCRIPTION
**Pull request by rymurr**
closes #310

### CODE REVIEW

File: app/ui/reports_query_hash.py [hunk 1]
```diff
@@ -105,7 +107,14 @@ def top_table():
...
         """
         )
-        st.dataframe(df, use_container_width=True)
+        df["Query Text"] = df.apply(
+            lambda row: f"{row['Query Id']}: {row['Query Text'][:10000]}"
+            if len(row["Query Text"]) > 10000
+            else row["Query Text"],
+            axis=1,
+        )
+        print(len(df.to_json()))
```
Comment (joshelser) [inline app/ui/reports_query_hash.py:116]: remove

File: app/ui/reports_query_hash.py [hunk 2]
```diff
@@ -105,7 +107,14 @@ def top_table():
 
         """
         )
-        st.dataframe(df, use_container_width=True)
+        df["Query Text"] = df.apply(
+            lambda row: f"{row['Query Id']}: {row['Query Text'][:10000]}"
```
Comment (joshelser) [inline app/ui/reports_query_hash.py:111]: Regular python `str`'s would do the correct thing if you did `"asdf"[:1000]` and the length of the string was less than the offset value. Does the dataframe act differently and throw some ArrayIndexOutOfBounds?

**Review by joshelser — CHANGES_REQUESTED** [review]

File: app/ui/reports_query_hash.py (hunk 2 above)
Comment (rymurr) [reply app/ui/reports_query_hash.py:111]: Row is a dict and the query text field is a str. Should behave just fine.

Comment (joshelser) [reply app/ui/reports_query_hash.py:111]: I missed the first half of the f-string in the if-clause. I thought it was just doing:
```
if len(queryText) > 1000:
  queryText[:1000]
else:
  queryText
```

**Review by joshelser — APPROVED** [review]

**Pull request merged**